frontend/app/routes/admin.js

```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  beforeModel() {
+    this.transitionTo('login');
```

Comment:
Isn't this will redirect to login in every case and the route is not accessible to anyone?

Reply:
right now there is no `super-admin` and `admin` exist, so I have added this as an temporary

Reply:
PR should not sent a temporary fix. Please look into the documentation more closely and send a complete fix.